modules/bitcoind-rpc-public-whitelist.nix

```diff
@@ -25,7 +25,6 @@
   "gettxoutsetinfo"
   "scantxoutset"
   "verifytxoutproof"
-  "waitfornewblock"
```

Comment:
I don't think that the whitelist is minimal for our services right now. If we wanted to do that we could remove quite a bit more. It consists of RPCs that are rather benign and `waitfornewblock` seems to fall in that category. So I'd rather keep it in. If we wanted to improve on this we could have individual whitelists per service.

Reply:
The main reason for removing it is that the old whitelist inspired https://github.com/romanz/electrs/issues/522 and the subsequent fix in electrs.
I'll let you decide.

Reply:
`waitfornewblock` is [not recommnended for public use](https://github.com/bitcoin/bitcoin/pull/10519#issuecomment-323094194). The old whitelist inspired https://github.com/romanz/electrs/issues/522 and the subsequent fix in electrs. I'll let you decide.

Reply:
There's a difference between public for nix-bitcoin services and public for the rest of the world. But I see that it's generally recommended to not use it. So it's fine by me to remove it (also, we're able to notice lower quality applications trying to use waitfornewblock).